cmd/ui/src/views/Explore/EdgeInfo/EdgeInfoContent.tsx

```diff
@@ -33,15 +39,66 @@ const EdgeInfoContent: FC<{ selectedEdge: NonNullable<SelectedEdge> }> = ({ sele
                 <>
                     {Object.entries(sections).map((section, index) => {
                         const Section = section[1];
+
+                        let handleOnChange = (label: string, isOpen: boolean) => {};
```

Comment:
Clever way of setting this function! Is there a period of time where `handleOnChange` is this function and not actually performing anything? I ask because as a user it may be confusing why Im clicking the accordion and nothing is happening. 

Alternatively, I wonder if a waiting indicator could be used while the needed values are loaded?

If the function is loaded quickly and there is no waiting period, totally disregard this comment!

Reply:
This `handleOnChange` function is only needed for the _Composition_ section and the collapsible section component will still open and close the accordion without this function being passed. I refactored this as per Wes' suggestion a couple of comments down so hopefully the intention is a little clearer but let me know if you still have any concerns.

Reply:
Ahh okay yeah that makes more sense, thanks for explaining that! The refactor makes it more clear as well :)